docs/_includes/ds6/button.html

```diff
@@ -96,7 +96,7 @@ <h4>Default Background (g202)</h4>
                 <span class="cta-btn__cell">
                     <span>Button</span>
                     <svg aria-hidden="true" class="cta-btn__icon" focusable="false" height="8" width="8">
-                        <use xlink:href="#icon-arrow-cta"></use>
+                        <use xlink:href="#icon-arrow-right"></use>
```

Comment:
I thought button was supposed to be using the `arrow-right-bold`

Reply:
Yes! Good spot! Will fix.